target/linux/mvebu/files-6.6/arch/arm64/boot/dts/marvell/armada-7040-rb5009upr.dts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "armada-7040-rb5009.dtsi"
+
+/ {
+	model = "MikroTik RB5009UPr";
+	compatible = "mikrotik,rb5009upr", "mikrotik,rb5009", "marvell,armada7040",
+		     "marvell,armada-ap806-quad", "marvell,armada-ap806";
+};
+
+&cp0_gpio1 {
+	enable-poe-power {
+		gpio-hog;
+		gpios = <8 GPIO_ACTIVE_HIGH>;
+		output-high;
+		line-name = "enable PoE power";
+	};
+};
+
+&cp0_pinctrl {
+	cp0_spi1_pins: cp0-spi1-pins {
+		marvell,pins = "mpp47", "mpp48", "mpp49", "mpp50";
+		marvell,function = "spi1";
+	};
+
+	/* PoE already enabled by enable-poe-power hog
```

Comment:
So what, this is just setting the pin to the GPIO function mode and you are not even calling it under pinctrl anywhere